scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1333,24 +1333,49 @@ static void task_uncharge_qrt(struct task_ctx *taskc)
 static void kick_idle_cpu(struct task_struct *p, struct layer *layer)
 {
 	const struct cpumask *idle_smtmask;
+	struct bpf_cpumask *cand_cpumask;
+	struct task_ctx *taskc;
+	struct cpu_ctx *cpuc;
 	s32 cpu;
 
 	if (!(idle_smtmask = scx_bpf_get_idle_smtmask()))
 		return;
-
+	
+	if (!(cpuc = lookup_cpu_ctx(-1)) || !(cand_cpumask = bpf_cpumask_create())) {
```

Comment:
This creates a new cpumask every enqueue, is that expensive?

Reply:
little idea but probably cheaper than the alternative (skipping later) when we need it. scoped creation to when we need it.